### PR TITLE
Promote t5 and variants

### PR DESCRIPTION
**Pull request by joecummings**

### CODE REVIEW

File: torchtext/models/__init__.py [hunk 1]
```diff
@@ -1 +1,2 @@
 from .roberta import *  # noqa: F401, F403
+from .t5 import *  # noqa: F401, F403
```
Comment (rshraga) [inline torchtext/models/__init__.py:2]: t5 not added yet so this will cause errors

Comment (joecummings) [reply torchtext/models/__init__.py:2]: It's added in this commit

Comment (joecummings) [reply torchtext/models/__init__.py:2]: OH nvm u right

File: test/torchtext_unittest/models/gpu_tests/models_gpu_test.py [hunk 2]
```diff
@@ -3,11 +3,12 @@
 import pytest
 import torch
 from torchtext_unittest.common.torchtext_test_case import TorchtextTestCase
-from torchtext_unittest.models.models_test_impl import BaseTestModels
+from torchtext_unittest.models.roberta_models_test_impl import RobertaBaseTestModels
```
Comment (rshraga) [inline test/torchtext_unittest/models/gpu_tests/models_gpu_test.py:6]: where is this added?

**Pull request merged**